app/views/visits/_form.html.haml

```diff
@@ -43,6 +43,9 @@
           .form-group.col-4.mb-0
             = f.label :effort_billing_qty, class: 'required'
             = f.text_field :effort_billing_qty, class: 'form-control'
+          .form-group.col-4{ id: 'select-t-qty' }
+            = f.label t(:calendars)[:pppv][:visits][:select_all_t]
```

Comment:
Can you set the `for` attribute (first parameter) on this label so clicking it will toggle the checkbox?